doc/configuration.md

```diff
@@ -1,108 +1,62 @@
 # Configuration Specification #
 
-The Ignition configuration is a JSON document conforming to the following
-specification, with **_italicized_** entries being optional:
+The Ignition configuration is a JSON document conforming to the following specification, with **_italicized_** entries being optional:
 
 - **ignitionVersion** (integer): the version number of the spec. Must be `1`.
-- **_storage_** (object): describes the desired state of the system's storage
-                          devices.
-  - **_disks_** (list of objects): the list of disks to be configured and their
-                                   options.
-    - **device** (string): the absolute path to the device. Devices are
-                           typically referenced by the /dev/disk/by-* symlinks.
-    - **_wipeTable_** (boolean): whether or not the partition tables shall be
-                                 wiped. When true, the partition tables are
-                                 erased before any further manipulation.
-                                 Otherwise, the existing entries are left
-                                 intact.
-    - **_partitions_** (list of objects): the list of partitions and their
-                                          configuration for this particular
-                                          disk.
+- **_storage_** (object): describes the desired state of the system's storage devices.
```

Comment:
`s/^- /* /` throughout these bullet lists. Most existing documents use the `*` marker for lists and we are going to enshrine that in the style guide.